taichi/struct/struct_llvm.h

```diff
@@ -10,9 +10,10 @@ class StructCompilerLLVM : public StructCompiler, public LLVMModuleBuilder {
  public:
   StructCompilerLLVM(Arch arch,
                      const CompileConfig *config,
-                     TaichiLLVMContext *tlctx);
+                     TaichiLLVMContext *tlctx,
+                     std::unique_ptr<llvm::Module> &&module = nullptr);
```

Comment:
nit: Can we avoid using default values here?

```suggestion
                     std::unique_ptr<llvm::Module> &&module);
```